scripts/security/sync_project_priorities.py

```
@@ -0,0 +1,293 @@
+# ---------------------------------------------------------------------------
+
+
+def _setup_logging(dry_run: bool) -> None:
+    logging.basicConfig(
+        level=logging.INFO,
+        format="%(asctime)s  %(levelname)-8s  %(message)s",
+        datefmt="%Y-%m-%d %H:%M:%S",
+        stream=sys.stdout,
+    )
+    if dry_run:
+        logging.info("DRY-RUN mode – no changes will be written to GitHub.")
```

Ruff (0.15.15)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)